src/quo2/screens/network_amount.cljs

```diff
@@ -0,0 +1,52 @@
+(ns quo2.screens.network-amount
+  (:require [quo.previews.preview :as preview]
+            [quo.react-native :as rn]
+            [quo2.components.network-amount :refer [network-amount]]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]))
+
+(def ^:private networks
+  [{:icon :main-icons2/ethereum :name "Mainnet"}
+   {:icon :main-icons2/arbitrum :name "Arbitrum"}
+   {:icon :main-icons2/optimism :name "Optimism"}
+   {:icon :main-icons2/zksync :name "zkSync"}])
+
+(defn- networks->options [networks]
+  (for [{:keys [name] :as network} networks]
+    {:key network
+     :value name}))
+
+(def ^:private descriptors
+  [{:label "ETH Value"
+    :key :eth-value
+    :type :text}
+   {:label "Network"
+    :key :network
+    :type :select
+    :options (networks->options networks)}])
+
+(def ^:private default-network
+  (first networks))
+
+(defn- state->opts [{:keys [eth-value network]
+                     :or {eth-value 5.123456}
+                     :as _state}]
```

Comment:
_state is unused

Reply:
@flexsurfer this is pure for readability, the `_`-prefix indicates the variable is unused, as suggested by clj-kondo.

Reply:
but an alias is optional, and if it's not used, why do we still want to keep it?

Reply:
it's not a big deal :) just wondering, seems like just individual preferences

Reply:
Not a big deal but I removed it anyway because this was only the second contribution where `:as _` was used in this repository, and it does not contribute that much in this case since the function name is clear.